Riot/Modules/MatrixKit/Utils/EventFormatter/HTMLFormatter.swift

```diff
@@ -51,6 +51,10 @@ class HTMLFormatter: NSObject {
             DTDefaultFontName: font.fontName,
             DTDefaultFontSize: font.pointSize,
             DTDefaultLinkDecoration: false,
+            // This fixes the issue where links are displayed in black
+            // However doesn't matter the value provided the color doesn't change
+            // From the default accent one
+            DTDefaultLinkColor: "",
```

Comment:
@aringenbach  this fixes the issue however this raises a question for the future... how do we change the default link color for all the app? It seems to be stuck to always be the green accent colour, however by speaking with @callumu , we would like to have the link's color to be blue as in the web version.
I can't seem to find any place where we change the default link color for the app, is it maybe a MatrixKit thing?